server/test/controllers/applicant/EligibilityAlertSettingsCalculatorTest.java

```diff
@@ -272,7 +272,8 @@ public void build_expected_eligibility_alert_settings_when_gating_is_on(ParamVal
             value.isApplicationEligible,
             value.isNorthStarEnabled, /* programId */
             value.pageHasSupplementalInformation,
-            1L);
+            1L,
+            ImmutableList.of());
```

Comment:
Probably should have a test that passes in something for this? (Definitely if you take my suggestion to move the transformation to the calculator)